docs/rules/prefer-promise-reject-errors.md

```diff
@@ -1,6 +1,6 @@
 # require using Error objects as Promise rejection reasons (prefer-promise-reject-errors)
 
-It is considered good practice to only pass instances of the built-in `Error` object to the `reject()` function for user-defined errors in Promises. `Error` objects automatically store a stack trace, which can be used to debug an error by determining where it came from. If a Promise is rejected with a non-`Error` value, it can be difficult to determine where the rejection occurred.
+It is considered a good practice to only pass instances of the built-in `Error` object to the `reject()` function for user-defined errors in Promises. `Error` objects automatically store a stack trace, which can be used to debug an error by determining where it came from. If a Promise is rejected with a non-`Error` value, it can be difficult to determine where the rejection occurred.
```

Comment:
I am not an expert in grammar, but as per Grammarly `a` should not be here - 

<img width="847" alt="Screenshot 2021-11-18 at 2 59 15 PM" src="https://user-images.githubusercontent.com/46647141/142388618-01533249-2448-4a45-9ea2-8902d405a776.png">

Reply:
Yes, “a” is incorrect here.